docs/jdbc-oceanbase-catalog.md

```diff
@@ -38,17 +38,18 @@ Check the relevant data source configuration in [data source properties](https:/
 If you use a JDBC catalog, you must provide `jdbc-url`, `jdbc-driver`, `jdbc-user` and `jdbc-password` to catalog properties.
 Besides the [common catalog properties](./gravitino-server-config.md#gravitino-catalog-properties-configuration), the OceanBase catalog has the following properties:
 
-| Configuration item   | Description                                                                                            | Default value | Required | Since Version |
-|----------------------|--------------------------------------------------------------------------------------------------------|---------------|----------|---------------|
-| `jdbc-url`           | JDBC URL for connecting to the database. For example, `jdbc:mysql://localhost:2881`                    | (none)        | Yes      | 0.3.0         |
-| `jdbc-driver`        | The driver of the JDBC connection. For example, `com.mysql.jdbc.Driver` or `com.mysql.cj.jdbc.Driver`. | (none)        | Yes      | 0.3.0         |
-| `jdbc-user`          | The JDBC user name.                                                                                    | (none)        | Yes      | 0.3.0         |
-| `jdbc-password`      | The JDBC password.                                                                                     | (none)        | Yes      | 0.3.0         |
-| `jdbc.pool.min-size` | The minimum number of connections in the pool. `2` by default.                                         | `2`           | No       | 0.3.0         |
-| `jdbc.pool.max-size` | The maximum number of connections in the pool. `10` by default.                                        | `10`          | No       | 0.3.0         |
+| Configuration item   | Description                                                                                                                           | Default value | Required | Since Version    |
+|----------------------|---------------------------------------------------------------------------------------------------------------------------------------|---------------|----------|------------------|
+| `jdbc-url`           | JDBC URL for connecting to the database. For example, `jdbc:mysql://localhost:2881`                                                   | (none)        | Yes      | 0.7.0-incubating |
+| `jdbc-driver`        | The driver of the JDBC connection. For example, `com.mysql.jdbc.Driver` or `com.mysql.cj.jdbc.Driver` or `com.oceanbase.jdbc.Driver`. | (none)        | Yes      | 0.7.0-incubating |
```

Comment:
If a user utilizes `com.oceanbase.jdbc.Driver`, will the MySQL protocol in the URL still work?

Reply:
Not work, Added `jdbc:oceanbase://localhost:2881` to jdbc-url example.

Reply:
Can the `com.oceanbase.jdbc.Driver` handle MySQL syntax? Have you confirmed this? Since currently the OB catalog implementation is based on MySQL driver

Reply:
Yes, I confirm. `com.oceanbase.jdbc.Driver` is based on MySQL protocol. Compared with `com.mysql.jdbc.Driver`, it supports both MySQL and Oracle syntax of OceanBase.